dnsmasq-plan/plan.sh

```diff
@@ -0,0 +1,31 @@
+pkg_name=dnsmasq
+pkg_origin=core
+pkg_version=2.80
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="Provides network infrastructure for small networks"
+pkg_upstream_url="http://www.thekelleys.org.uk/dnsmasq/doc.html"
+pkg_license=("GPL")
```

Comment:
Could you please update this to `GPL-2.0-or-later` to match the SPDX short-description string. https://spdx.org/licenses/GPL-2.0-or-later.html